scripts/build/build.ps1

```diff
@@ -33,8 +33,8 @@ else {
 header "Compiling"
 _ "$RepoRoot\scripts\compile\compile.ps1" @("$Configuration")
 
-header "Running Tests"
-_ "$RepoRoot\scripts\test\runtests.ps1"
+header "Running Tests on Stage 2"
+_ "$RepoRoot\scripts\test\runtests.ps1" @("$Stage2Dir")
```

Comment:
Lets do something similar to the unix scripts. All the required environment variables are set before calling runtests.sh - https://github.com/dotnet/cli/blob/master/scripts/build/build.sh#L42

This way we maintain consistency between platforms and keep runtests.ps1 agnostic of which dotnet tools it is testing. Example - In the future we will want to use the runtests.ps1 to test MSI installations.